Dockerfile

```diff
@@ -1,16 +1,21 @@
 FROM alpine:3.13
 
 ENV TERRAFORM_VERSION=0.13.6
+ARG TARGETPLATFORM
```

Comment:
We should have some default

Reply:
I'd leave this as is since this will yield an error whenever we try to use an "unsupported platform" (the one that's not in the if below).

Secondly I'd add a comment as to where `TARGETPLATFORM` is coming from: e.g.  https://docs.docker.com/engine/reference/commandline/buildx_build/#platform

Reply:
I dont get it, if we put default it'll raise error anyway if `TARGETPLATFORM` will be invalid

Reply:
Don't see a relation between those

Reply:
Since `TARGETPLATFORM` is coming from the docker itself why would you specify a default?

Reply:
Thats fair :+1:

Reply:
A default for `TARGETPLATFORM`? This is not a build arg for the end user to set, it is set by Docker engine - https://docs.docker.com/engine/reference/builder/#automatic-platform-args-in-the-global-scope. I don't see a point in setting a default on it.